### PR TITLE
jupyter: Update episode, create overall outline

### PR DESCRIPTION
**Pull request by rkdarst**
- Create a workable outline, that gets from exercise to exercise, and
  contains a lot of the key points.
- Currently missing some screenshots (somehow flameshot stopped
  working for me... couldn't add text)
- Review:
  - General look, will iterate.  Could be merged now or later.
  - While there isn't a lot of text, it's not in bullet points so
    seems kind of text-heavy.  That should be improved somehow -
    ideas for general best practices?
  - Needs some thought about how to do the demos - is this very
    notebook edited during the lesson?  Make new notebook?

### CODE REVIEW

**Comment by bast** [comment]
How do others couple nbdime and GitHub? Do you diff these locally?

**Review by annefou — COMMENTED** [review]
What I miss in this section is a bit more about how to exploit jupyter notebooks:
- prototyping, testing new ideas
- publication ready notebooks
- teaching, providing examples that can be easily read on github (with embedded explanation) as notebooks are rendered in github.

Would it make sense to add a seciotn on these aspects?

**Comment by annefou** [comment]
> How do others couple nbdime and GitHub? Do you diff these locally?

I use my local jupyterLab with git extension.

File: jupyter.ipynb [hunk 1]
```diff
@@ -42,39 +45,37 @@
...
+    "      - If you are starting from the command line, you should navigate to the directory you want to use first.\n",
     "\n",
-    "   2. Create a notebook file.\n",
+    "   2. Create a Python 3 notebook file.  Save it.  In the next section, you will add stuff to it.\n",
     "\n",
-    "   3. What directory does Jupyter show you.  How can you access\n",
-    "      files in other directories?\n",
+    "   3. Open a terminal.  Run `python3 -V` to check your Python version.\n",
+    "   \n",
+    "   4. Create a new file.  Rename it to `test.py` (to rename, right click on the filename either on the left or on the tab above, or use the File menu).\n",
```
[the remark this answers is not in the document]
Comment (rkdarst) [reply jupyter.ipynb:73]: Hm, I thought better to add it as a gentle introduction, something that is easy to expect people to try when starting off.  We can possible see problems for day 2.  Can you think of some other very easy exercise at this point?

[the remark this answers is not in the document]
Comment (annefou) [reply jupyter.ipynb:73]: Why not adding an exiercise on the usage of markdown cells. Can also be useful for hackMD.

Check what notebooks are running and learn to "shutdown" them.

[the remark this answers is not in the document]
Comment (annefou) [reply jupyter.ipynb:73]: Can also create a mardown file (instead of `test.py`)

**Review by annefou — APPROVED** [review]

**Comment by annefou** [comment]
Great. I think it is good to merge it now.

**Pull request merged**